templates/docs/layouts/documentation.md

```diff
@@ -0,0 +1,86 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Documentation | Layouts
+---
+
+## Documentation layout
+
+<hr>
+
+### Structure
+
+The documentation page layout can be built using the Vanilla grid classes and common components. It consists of 3 horizontal areas that span the entire fixed width of the grid: header, content, footer.
+
+![Documentation layout structure](https://assets.ubuntu.com/v1/cdf09557-Documentation+layout%401x.svg)
+
+At the large breakpoint, the content area is further divided into an aside (3 columns) and a main content area (9 columns).
+
+At smaller breakpoints, the aside is moved offscreen and shown / hidden using a toggle.
+
+#### Header
+
+The header with the main navigation, built with the [navigation component](/docs/patterns/navigation#global-navigation).
+
+Style and contents of the documentation main navigation should be consistent with rest of the site.
+
+##### Search
+
+Documentation pages may have an optional search box in the main navigation.
+
+Alternatively search can be added in a full-width area under the top navigation, but above the aside and main content in a [strip component](/docs/patterns/strip) with grid row inside. Usually, it would be shallow light strip (`.p-strip--light is-shallow`), but the specific styling can be customised to match the site branding or other design requirements.
+
+#### Content area
+
+The content area is implemented as a regular strip (`.p-strip`) with a grid row (`.row`) inside. Within the standard Vanilla 12 column grid, 3 of the columns are reserved for the side navigation (`.col-3`) with the rest of the row width (9 columns, `.col-9`) is deadicated to the main documentation content.
+
+Sidebar should contain only the [side navigation component](/docs/patterns/navigation#side-navigation) with a list of all documentation pages. Grouping and nesting of navigation items should be used to build the logical structure of documentation navigation. Side navigation component has built-in responsive functionality which makes the sidebar expandable on small screens.
```

Comment:
Did we make the decision to go with sidebar vs aside? I'm ok either way, just checking if we're naming it consistently.